### ISSUE TITLE
Adding a socials section to the readme and adding badges to it

### ISSUE THREAD

**Issue by 1Shubham7** [1Shubham7/code-buddy#12]

**Comment by andrewmuhn** [comment]
When you say socials do you mean a contact section with socials of contributors? If so which socials would you like to see?

**Comment by 1Shubham7** [comment]
No, I mean my social media account links. you can add some badges and add my links to it, links and badges are avalable on my profile itself you just have to copy them. Do you want to work on this issue ?

**Comment by andrewmuhn** [comment]
Yeah I'll work on it.
